lxd/db/instances.go

```diff
@@ -1108,6 +1108,33 @@ func (c *Cluster) GetInstancePool(project, instanceName string) (string, error)
 	return poolName, err
 }
 
+// DeleteReadyStateFromLocalInstances deletes the volatile.last_state.ready config key
+// from all local instances.
+func (c *Cluster) DeleteReadyStateFromLocalInstances() error {
+	err := c.Transaction(context.TODO(), func(ctx context.Context, tx *ClusterTx) error {
```

Comment:
Can just do `return c.Transation(...` here as you're not wrapping error context.